### PR TITLE
FEAT(backend): create intial spring boot project

### PR DESCRIPTION
**Pull request by Paw565pl**

### CODE REVIEW

**Comment by Paw565pl** [comment]
I have yet to add our backend to docker compose.

**Comment by Paw565pl** [comment]
Everything should be ready now.

**Review by DamianRogalskiUG — APPROVED** [review]

**Review by GracjanKoscinski — COMMENTED** [review]
LGTM

File: backend/src/main/java/it/compare/backend/core/exception_handler/ErrorResponse.java
Comment (DamianRogalskiUG) [inline backend/src/main/java/it/compare/backend/core/exception_handler/ErrorResponse.java:1]: Package name does not match to the standard pattern, should not contain dash

File: backend/src/main/java/it/compare/backend/core/exception_handler/GlobalExceptionHandler.java
Comment (DamianRogalskiUG) [inline backend/src/main/java/it/compare/backend/core/exception_handler/GlobalExceptionHandler.java:1]: Same story with package name pattern here

**Review by GracjanKoscinski — APPROVED** [review]
LGTM

**Review by DamianRogalskiUG — APPROVED** [review]

**Pull request merged**